### PR TITLE
Not allowing to create a post dated event

### PR DESCRIPTION
**Pull request by vubo**
Issue https://github.com/systers/vms/issues/78

image

### CODE REVIEW

File: vms/event/templates/event/create.html [hunk 1]
```diff
@@ -1,7 +1,22 @@
 {% extends "administrator/settings.html" %}
 
+
+
 {% block setting_content %}
-    <div class="spacer"></div>
+<br>
```
Comment (vubo) [inline vms/event/templates/event/create.html]: Show alert message

File: vms/event/views.py [hunk 2]
```diff
@@ -28,12 +29,18 @@ def is_admin(request):
...
 @login_required
 def create(request):
+    
     if is_admin(request):
         if request.method == 'POST':
             form = EventForm(request.POST)
             if form.is_valid():
-                form.save()
-                return HttpResponseRedirect(reverse('event:list'))
+                start_date = form.cleaned_data['start_date']
```
Comment (vubo) [inline vms/event/views.py]: Check date

File: vms/volunteer/models.py [hunk 3]
```diff
@@ -111,7 +111,8 @@ class Volunteer(models.Model):
         )
     reminder_days = models.IntegerField(
         default=1,
-        validators=[MaxValueValidator(50), MinValueValidator(1)]
+        validators=[MaxValueValidator(50), MinValueValidator(1)],
+        blank=True
```
Comment (vubo) [inline vms/volunteer/models.py]: A small fix for the https://github.com/systers/vms/pull/93

File: vms/event/views.py [hunk 4]
```diff
@@ -28,12 +29,18 @@ def is_admin(request):
...
+                if start_date < (datetime.date.today() - datetime.timedelta(days=1)):
+                    messages.add_message(request, messages.INFO, 'You can not create a post dated event')
```
Comment (willingc) [inline vms/event/views.py]: @vubo Let's give a more helpful error message to the end user :) Perhaps, "Start date should be tomorrow's date or later."

**Comment by willingc** [comment]
@vubo Looks good. Just one minor request to change the error message to be more user friendly.

@tapasweni-pathak If you concur that all if fine, please merge. Thanks :)

**Comment by vubo** [comment]
@willingc thank you, I did it. Comitted and rebased :)

**Comment by tapasweni-pathak** [comment]
@willingc Done. :)

@vubo :+1:

**Pull request merged**